services/pama-imaging.js

```diff
@@ -24,42 +40,62 @@ class Reasons {
     return true;
   }
 
-  constructor(yes, no) {
-    this.yes = yes.map(x => new Set(x));
-    this.no = no.map(x => new Set(x));
+  constructor(appropriate, notAppropriate) {
+    this.appropriate = appropriate.map(x => new Set(x));
+    this.notAppropriate = notAppropriate.map(x => new Set(x));
   }
 
   getRating(reasons) {
-    if (this.yes.filter(s => Reasons.covers(s, reasons)).length) {
+    if (this.appropriate.filter(s => Reasons.covers(s, reasons)).length) {
       return 'appropriate';
     }
-    if (this.no.filter(s => Reasons.covers(s, reasons)).length) {
+    if (this.notAppropriate.filter(s => Reasons.covers(s, reasons)).length) {
       return 'not-appropriate';
     }
     return 'no-guidelines-apply';
   }
 }
 
 const cptReasons = {
-  '1234': new Reasons([['1']], [['2', '3']]), // testing only
-  '70450': new Reasons([['25064002', '423341008']], []),
-  '70544': new Reasons([], []),
-  '72133': new Reasons([], [['279039007']]),
-  '75561': new Reasons([['13213009']], []),
+  '1234': new Reasons([['1']], [['2', '3']]),
+  [CPT.CT_HEAD_NO_CONTRAST]: new Reasons([[SNOMED.HEADACHE, SNOMED.OPTIC_DISC_EDEMA]], []),
+  [CPT.MRA_HEAD]: new Reasons([], []),
+  [CPT.CTA_WITH_CONTRAST]: new Reasons([], [[SNOMED.CONGENITAL_HEART_DISEASE]]),
+  [CPT.LUMBAR_SPINE_CT]: new Reasons([], [[SNOMED.LOW_BACK_PAIN]]),
+  [CPT.CARDIAC_MRI]: new Reasons([[SNOMED.CONGENITAL_HEART_DISEASE]], []),
+};
+
+const CARD_TEMPLATES = {
+  [SNOMED.CONGENITAL_HEART_DISEASE]: [{
```

Comment:
Is your card recommendation coming back with a pama rating? i.e. if it's being suggested, it's specified as being 'appropriate'?